manim/mobject/mobject.py

```diff
@@ -2039,40 +2040,236 @@ def construct(self):
+            vertical direction, a tuple of two values can be given. The first value
+            containing defines the buffer in horizontal, the second the buffer in vertical
+            direction.
+        cell_alignment
+            The way each submobject is aligned in its grid cell.
+        row_alignments
+            The way each row of submobjects (from top to bottom) is aligned vertically.
+            If defined the given string must only contain the following characters: "u"
+            (up), "c" (centered) and "d" (down).
+        col_alignments
+            The way each column of submobjects (from left to righht) is aligned horizontally.
+            If defined the given string must only contain the following characters: "l"
+            (left), "c" (centered) and "r" (right).
+        row_heights
+            Specific heights for certain rows (from top to bottom). The given list can contain
+            `None`. In that case the corresponding row will fit its height automatically based
+            on the highest element in that row.
+        col_widths
+            Specific widths for certain columns (from left to right). The given list can contain
+            `None`. In that case the corresponding column will fit its width automatically based
+            on the widest element in that column.
+        flow_order
+            The order in which the grid gets filled with the submobjects. Can be one of the
+            following values: "rd", "dr", "ld", "dl", "ru", "ur", "lu", "ul". For example,
+            "dr" (down right) means: Fill the table column by column downwards, filling the
+            columns rigthwards. "u" and "l" stand for upwards and leftwards.
+
+        .. note:: If either :code:`rows` or :code:`cols` is not defined, it is set to the length of either
+            :code:`row_alignments` / :code:`col_alignments` or :code:`row_heights` / :code:`col_widths` if
+            one of those is defined. If only one of :code:`rows` and :code:`cols` can be determined
+            like this, the other will be choosen big enough to fit all submobjects.
+            If neither :code:`cols` nor :code:`rows` is set implicitly, they will be choosen to be
+            about the same, tending towards :code:`cols > rows` (simply because videos are wider
+            then they are high).
+
+        .. note:: If both :code:`cell_alignment` and either :code:`row_alignments` or :code:`col_alignments` is
+            defined the latter has higher priority in its direction.
+
+
+        Returns
+        -------
+        Mobject
+            The mobject.
+
+        Raises
+        ------
+        ValueError
+            If rows and cols are choosen to small to fit all submobjetcs.
+        ValueError
+            :code:`cols`, :code:`col_alignments` and :code:`col_widths` or :code:`rows`,
+            :code:`row_alignments` and :code:`row_heights` have mismatching sizes.
```

Comment:
Just to be consistent with the previous ValueError
```suggestion
        ValueError
            If :code:`cols`, :code:`col_alignments` and :code:`col_widths` or :code:`rows`,
            :code:`row_alignments` and :code:`row_heights` have mismatching sizes.
```